test/Faker/Provider/nl_NL/PersonTest.php

```diff
@@ -18,7 +18,7 @@ public function testGenerateValidIdNumber()
         $sum = -1 * $idNumber % 10;
 
         for ($multiplier = 2; $idNumber > 0; ++$multiplier) {
-            $val = ($idNumber /= 10) % 10;
+            $val = (int) ($idNumber /= 10) % 10;
```

Comment:
```
  1x: Implicit conversion from float 67899705.6 to int loses precision
    1x in PersonTest::testGenerateValidIdNumber from Faker\Test\Provider\nl_NL
```
`$idNumber /= 10` generates a float which is then used with modulo, but modulo converts everything to integer first:
> Operands of modulo are converted to int before processing. For floating-point modulo, see fmod().
https://www.php.net/manual/en/language.operators.arithmetic.php

Reply:
Thanks for the clear explanation!